types/graphql-schema-linter/lib/configuration/index.d.ts

```diff
@@ -0,0 +1,8 @@
+import { ValidationContext, ASTKindToNode, Visitor, GraphQLError } from 'graphql';
```

Comment:
In the source, there is no `configuration/index.js`, but rather just `configuration.js`. Since an import of `graphql-schema-linter/lib/configuration/index` would fail at runtime, I would recommend moving/renaming this file to `lib/configuration.d.ts`.

Reply:
The same seems to be true of *every* file under `lib/**`.